templates/rustdoc/topbar.html

```diff
@@ -84,6 +84,14 @@
                                 </a>
                             </li>
 
+                            {# A link to the release's source view #}
+                            <li class="pure-menu-item">
```

Comment:
Why did you move this item here?

Reply:
In the discussion in #1205 , we realized that there was a need for some additional space in the navbar to accommodate the combination of long crate names and warnings. The "Source" menu item was identified as one to relocate into the crate menu.